examples/arm/aot_arm_compiler.py

```diff
@@ -273,7 +273,7 @@ def get_compile_spec(
                 target,
                 system_config="Ethos_U55_High_End_Embedded",
                 memory_mode="Shared_Sram",
-                extra_flags="--debug-force-regor --output-format=raw",
+                extra_flags="--debug-force-regor --output-format=raw --verbose-operators --verbose-cycle-estimate",
```

Comment:
@zingo ^^^ 

@digantdesai This is the revert. Merging.

Reply:
> Nit: Add these optionally behind a `verbose` flag?

Good idea, might do that. Up until this version vela always printed this but started to hide it behind this flag now. we have some scripts picking it up but in our CI but it could absolutely migrate to a flag. We have an task/idea to rework the flow and separate testing from user examples so it will fit perfectly into the idea we have.